aocd/models.py

```diff
@@ -221,6 +225,17 @@ def answer_b(self, val):
             return
         self._submit(value=val, part="b")
 
+    @property
+    def answered_b(self):
+        return bool(getattr(self, "answer_b", None))
+
+    def answered(self, part):
+        if part == "a":
+            return bool(getattr(self, "answer_a", None))
+        if part == "b":
+            return bool(getattr(self, "answer_b", None))
+        raise AttributeError
```

Comment:
```suggestion
        raise AocdError('part must be "a" or "b"')
```

An `AttributeError` is not really appropriate here, since using `puzzle.answered("c")` is not actually accessing any incorrect attribute.